test/integration/narrow/app/fuzzySearchService.test.js

```diff
@@ -37,72 +47,32 @@ describe('testing fuzzySearchService /paymentdata', () => {
       ]
     }
     const mockDb = jest.spyOn(PaymentDataModel, 'findAll')
-    mockDb.mockResolvedValue(mockData)
-    const result = await getPaymentData(searchString, limit, offset, sortBy)
-    expect(result.count).toEqual(2)
-    expect(result).toEqual(expectedData)
-  })
-
-  test('GET /paymentdata returns right paginated data', async () => {
-    const { getPaymentData } = require('../../../../app/services/fuzzySearchService')
-    const searchString = 'fa'
-    const limit = 10
-    const offset = 0
-    const sortBy = null
-    const mockDb = jest.spyOn(PaymentDataModel, 'findAll')
     mockDb.mockResolvedValue(paymentestdata)
-    const result = await getPaymentData(searchString, limit, offset, sortBy)
-    expect(result.count).toBeGreaterThan(10)
-    expect(result.rows.length).toEqual(10)
-    const result2 = await getPaymentData(searchString, 5, offset, sortBy)
-    expect(result2.count).toEqual(11)
-    expect(result2.rows.length).toEqual(5)
-  })
 
-  test('GET /paymentdata returns second page data', async () => {
-    const { getPaymentData } = require('../../../../app/services/fuzzySearchService')
-    const searchString = 'fa'
-    const limit = 10
```

Comment:
Not 100% sure what you mean but these test criteria do change between tests. For example within test labelled 'GET /paymentdata returns right paginated data', we set the limit (page size) to 5 because searchString 'fa' only return 11 results and we want to test that we get different results based on the offset.

Reply:
I was saying this was already working with limit 10 and the results were tested successfully, but changing to limit 5 is not making much of difference.